src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -174,6 +174,10 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
     @Restricted(NoExternalUse.class)
     public static boolean USE_WATCHING = Boolean.parseBoolean(System.getProperty(DurableTaskStep.class.getName() + ".USE_WATCHING", Main.isUnitTest ? "true" : /* JENKINS-52165 turn back on by default */ "false"));
 
+    /** How many seconds to wait before interrupting remote calls and before forcing cleanup when the step is stopped */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "public & mutable for script console access")
+    public static long REMOTE_TIMEOUT = Integer.parseInt(System.getProperty(DurableTaskStep.class.getName() + ".REMOTE_TIMEOUT", "10"));
```

Comment:
Any thoughts on increasing the default value to maybe 20 seconds? The user in Jira ended up using 60 seconds, which feels a little high as a default value, but I think increasing the default would be a nice way to help users without them needing to investigate and discover this property, and 20 seconds seems reasonable even for users not seeing the specific issue.

Reply:
20 sounds sane to me.